DSACMS/iv-cbv-payroll
# FFS-2211: guardrails for nil pinwheel_event

## [FFS-2211](https://jiraent.cms.gov/browse/FFS-2211)

## Changes
Updated synchronizations redirect logic to fail gracefully if we can't find a pinwheel_account
Updated turbo-rails so we incorporate a fix that will help our webserver not crash when this happens

## Context for reviewers
This is part of a response to a downtime event we encountered on 12/5. More details in the Jira ticket.

## Verification Instructions
0. Go through your regular workflow, but pay attention when you get to the synchronizations step (spinning icons when loading from pinwheel). Copy the URL from this step for later.
1. Verify the workflow succeeds. There should be no regression in behavior.
2. Paste the URL you copied but change the ID in the URL to something slightly different (and likely to not be real). When you navigate to this page, you should see our nice "the sync failed" screen. Previously, you would have seen an ugly Rails application error.

## Acceptance testing


  * This change will not affect the user experience (bugfix, dependency updates, etc.)
  * This change can be verified visually via screenshots attached below or by sending a link to a local development environment to the acceptance tester
  * Acceptance testing should be done by **design** for visual changes, **product** for behavior/logic changes, **or both** for changes that impact both.
  * This change is hard to test locally, so we'll test it in the demo environment (deployed automatically after merge.)
  * Make sure to notify the team once this PR is merged so we don't inadvertently deploy the unaccepted change to production. (e.g. `:alert: Deploy block! @ffs-eng I just merged PR [#123] and will be doing acceptance testing in demo - please don't deploy until I'm finished!`)
